### ISSUE TITLE
popover far from button

### ISSUE THREAD

**Issue by GMounir** [euvl/vue-js-popover#28]
Thank you for this plugin 👍 .

Why popover not close to button

button(v-popover.left="{ name: 'foo' }") Toggle popover
popover(name="foo") Hello

**Comment by GMounir** [comment]
Do i have to add position to this class?

	div[data-popover="tooltip"] {
		top: -5px !important;
		left: 20% !important;
	}

**Comment by quodos** [comment]
This seems to be a problem with parent elements that have position:relative. Anyone has a fix already?

**Comment by affinitycloud** [comment]
Yup! Getting the same thing. It's absolutely miles away. In fact, it's off-screen.

**Comment by stevenharman** [comment]
Is this a duplicate of #27, which has been fixed by #34?

**Comment by mikerodham** [comment]
I'm not using position relative, in fact just display: block; and it's on the other side of the page.

**Comment by HJBWeb** [comment]
This also seems to be happening when the parent is position:fixed.

**Comment by JosNun** [comment]
@stevenharman This is still happening for me on v 1.1.8. Any suggestions?
